x/ecocredit/client/query.go

```diff
@@ -439,17 +413,15 @@ regen q ecocredit balance C01-001-20200101-20210101-001 regen1r9pl9gvr56kmclgkpj
 	})
 }
 
-// QuerySupplyCmd returns a query command that retrieves the tradable and
+// QueryBatchSupplyCmd returns a query command that retrieves the tradable and
 // retired supply of credits for a given credit batch.
-func QuerySupplyCmd() *cobra.Command {
+func QueryBatchSupplyCmd() *cobra.Command {
 	return qflags(&cobra.Command{
-		Use:   "supply [batch_denom]",
-		Short: "Retrieve the tradable and retired supply of the credit batch",
-		Long:  "Retrieve the tradable and retired supply of the credit batch",
-		Example: `
-regen q ecocredit supply C01-001-20200101-20210101-001
-		`,
-		Args: cobra.ExactArgs(1),
+		Use:     "batch-supply [batch-denom]",
```

Comment:
Although `supply` corresponds with the ecocredit module, we have basket and marketplace submodule commands that exist alongside the ecocredit commands so `batch-supply` made more sense.

I think in general we should try to match the name of the command with the message or query name but we might need to rethink client architecture with submodules.